vulnerabilities/importers/github.py

```diff
@@ -146,189 +122,339 @@
     "-class.-jw.util.version.Version-",
 }
 
+ECOSYSTEM_PKG_TYPE_MAPPING = {
+    "MAVEN": "maven",
+    "NUGET": "nuget",
+    "COMPOSER": "composer",
+    "PIP": "pypi",
+    "RUBYGEMS": "gem",
+    "GO": "golang",
+}
+
+ECOSYSTEM_VERSION_RANGE_MAPPING = {
+    "MAVEN": MavenVersionRange,
+    "NUGET": NugetVersionRange,
+    "COMPOSER": ComposerVersionRange,
+    "PIP": PypiVersionRange,
+    "RUBYGEMS": GemVersionRange,
+    "GO": GolangVersionRange,
+}
+
+PKG_TYPE_ECOSYSTEM_MAPPING = {value: key for (key, value) in ECOSYSTEM_PKG_TYPE_MAPPING.items()}
+
+QUERY = """
```

Comment:
What about this more verbose but self-explanatory?
```suggestion
GRAPHQL_VULNERBILITY_QUERY_TEMPLATE = """
```